yaml/pois.yaml

```diff
@@ -288,7 +288,11 @@ filters:
     min_zoom: 18
     output: {kind: toilets}
   - filter: {barrier: gate}
-    min_zoom: 15
+    min_zoom: CASE WHEN mz_get_highway_level_point(osm_id) = 3 THEN 14
+        WHEN mz_get_highway_level_point(osm_id) = 2 THEN 15
+        WHEN mz_get_highway_level_point(osm_id) = 1 THEN 16
```

Comment:
I worry that this might result in running this function 3 times for most instances, and we're expecting the function to be moderately expensive as it joins back to the `planet_osm_ways` table. Would it be okay to re-write this as `17 - COALESCE(mz_get_highway_level_point(osm_id), 0)`? (The `COALESCE` is just there to make sure a number is used, as the function can sometimes return `NULL`).